pkg/cbpf/jmp_instructions.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cbpf
+
+import (
+	pb "buzzer/proto/cbpf_go_proto"
+)
+
+func newJmpInstruction(oc pb.JmpOperationCode, jmpTrue int32,
+	jmpFalse int32, fieldK int32) *pb.Instruction {
+
+	opcode := int32(0)
+
+	// The 3 least significant bits are the instruction class.
+	opcode |= (int32(pb.InsClass_InsClassJmp) & 0x07)
+
+	// The fourth bit is the source operand.
+	opcode |= (int32(pb.SrcOperand_Immediate) & 0x08)
```

Comment:
Does this mean the jumps are always immediate?

Reply:
No, the implementation is incorrect. I'll add support for registers